Sources/graphql-swift/codegen/schemaGenerator.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+struct GraphqlSchemaRequest: Codable {
+    let query: String
+}
+
+public struct SchemaTypeGenerator {
+    let client: GraphqlClient!
+    
+    public init(_ client: GraphqlClient) {
```

Comment:
```suggestion
    public init(client: GraphqlClient) {
```

Also note that this explicit `init` is only needed if you’re importing this module from another module. Otherwise you can use the default synthesized (internal) `init`.

Reply:
ohhhh - good call, thank you!!!